pkgs/development/libraries/openssl/default.nix

```diff
@@ -58,6 +59,9 @@ stdenv.mkDerivation rec {
 
     # remove dependency on Perl at runtime
     rm -r $out/etc/ssl/misc $out/bin/c_rehash
+
+    # configure the default trust store
+    ${optionalString (defaultCertificate != null) "ln -s ${defaultCertificate} $out/etc/ssl/cert.pem"}
```

Comment:
Instead of making a symlink, why not just override `X509_CERT_DIR` at compile time?

Also, do we actually want to have this configurable? I can't think of a good reason to change the default.